src/init/autoApplyCustomizations.ts

```diff
@@ -6,7 +6,7 @@ export function manageAutoApplyCustomizations(
   userConfig: IVSIcons,
   applyCustomizationCommand: () => void): void {
   if (!isNewVersion) { return; }
-  const propObj = packageJson.contributes.configuration.properties || {};
+  const propObj: any = packageJson.contributes.configuration.properties as object;
```

Comment:
What's the use of `as object` if you have notated `propObj` as any? In fact, the any is implicit in `packageJson.contributes.configuration.properties`.